packages/rancher-kiali-server/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: https://kiali.org/helm-charts/kiali-server-1.29.0.tgz
-packageVersion: 00
-releaseCandidateVersion: 01
+packageVersion: 01
+releaseCandidateVersion: 00
```

Comment:
000 and 001 are both invalid, so this specific change doesn't matter. 
Wouldn't helm complain whether is sub-chart or not ? And CRD chart is top-level chart so that is null point anyway.

Reply:
If this is a problem we could literally change the kiali version getting installed since the user never see's it I don't think. And then once they release `.1` we just bump the package version.

Reply:
000 is valid since it's not a trailing zero iirc. @mattfarina may be able to confirm

Reply:
Helm won't complain for a subchart, but the CRD chart will definitely cause complaints. It might make sense to bump kiali's patch version to 100 IMO.

Reply:
does this also apply for jaeger? I will get kiali updated.

Reply:
yes good catch @brendarearden, can we just bump the patch version since it's a local chart? https://github.com/rancher/charts/blob/54da22aa720ef05656768cd0d8767f0e5916a042/packages/rancher-tracing/charts/Chart.yaml#L5

Reply:
I used Matt's tool to check: 
```
$ semver-isvalid 1.2.000
Illegal leading 0 found in "patch" part
Invalid Semantic Version: Version segment starts with 0. For more information see https://semver.org

$ semver-isvalid 1.2.001
Illegal leading 0 found in "patch" part
Invalid Semantic Version: Version segment starts with 0. For more information see https://semver.org
```

As far as what to do with kiali and jaeger for now. We will have to upgrade helm with a rancher release, and there will likely be a kiali and istio version in that rancher release. Jaeger may be less likely.

Reply:
Okay, I added a commit just for the kiali and jaeger version updates so its ready for -re-review